nodejs/week4/homework-template/src/backend/api/reservation.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
```

Comment:
I would be consistent with the file naming and call this file `reservations.js`.